R/runtime-caches.R

```diff
@@ -0,0 +1,73 @@
+#' Get runtime caches
+#' 
+#' View the runtime caches on a Connect server.
+#' 
+#' @param client A `Connect` object.
+#' 
+#' @returns A tibble of runtime caches on the server, showing `language`,
+#' `version` and `image_name`. For Connect servers not using off-host execution,
+#' `image_name` is `"Local"`.
+#' 
+#' @examples
+#' \dontrun{
+#' client <- connect()
+#' get_runtime_caches(client)
+#' }
+#' 
+#' @family server management functions
+#' @export
+get_runtime_caches <- function(client) {
+  validate_R6_class(client, "Connect")
+  res <- client$GET(v1_url("system/caches/runtime"))
+  purrr::map_dfr(res$caches, ~.x)
+}
+
+#' Delete a runtime cache
+#' 
+#' Delete a runtime cache from a Connect server.
+#' 
+#' @param client A `Connect` object.
+#' @param language The language of the cache, either "R" or "Python".
+#' @param version The version of the cache, e.g. "3.6.3".
```

Comment:
Let's update to a recent version

```suggestion
#' @param version The version of the cache, e.g. "4.3.3".
```